test/cpp/util/grpc_tool.cc

```diff
@@ -35,6 +27,14 @@
 #include <grpcpp/security/credentials.h>
 #include <grpcpp/support/string_ref.h>
 
+#include <cstdio>
```

Comment:
I think Vijay mentioned we may not be able to move these?

Reply:
This change occurred after I ran clang-format and it looks like the formatting checks have passed, so moving them seems fine.